image/create.go

```diff
@@ -30,6 +30,7 @@ import (
 	"encoding/binary"
 	"io/ioutil"
 	"math/big"
+    "encoding/base64"
```

Comment:
`go` uses tabs for formatting, you can use `go fmt` to automatically do it. I use `go-vim` plugin (if you are a vim user!).

Reply:
that is cruel, using tabs here and not elsewhere =D

Reply:
fixed